src/components/grid-aware/Footer/SubscriptionBlock/SubscriptionBlock.module.css

```diff
@@ -0,0 +1,59 @@
+/* Background bleed */
+
+.bleedWrapper {
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Top Row */
+
+.subscriptionBlock {
+  background-color: var(--color-black);
+  display: flex;
+  justify-content: center;
+  padding: 35px;
```

Comment:
In mobile, this padding isn't right. Can you override this in a media query?

Reply:
ok

Reply:
padding: 44px 42px 43px; in mobile